public/js/form_helpers.js

```diff
@@ -12,7 +12,13 @@
  */
 
 $(document).ready(function(){
-    initMask();
```

Comment:
Not sure why you removed this. Needs to be here for the masks to be initialized.